README.md

```diff
@@ -1,368 +1,257 @@
+
+  ```bash
+  foreman start -f Procfile.dev
   ```
-1. See customization of configuration options below.  
 
-### Additional Steps For Server Rendering (option `prerender` shown below)
-See the next section for a sample webpack.server.rails.config.js.
-1. Expose your server globals like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/app/startup/serverGlobals.jsx#L7)
-   
-   ```javascript
-   import App from './ServerApp';
-   global.App = App;
-   ```
-2. Make the server globals file an entry point in your webpack config, like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.server.rails.config.js#L7)
-
-   ```javascript
-   entry: ['./app/startup/serverGlobals'],
-   ```
-3. Ensure the name of your ouput file (shown [here](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.server.rails.config.js#L9)) of your server bundle corresponds to the configuration of the gem. The default path is `app/assets/javascripts/generated`. See below for customization of configuration variables.
-4. Expose `React` in your webpack config, like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.server.rails.config.js#L23)
+## Generator Options
+Run `rails generate react_on_rails:install --help` for descriptions of all available options:
 
-```javascript
-      { test: require.resolve('react'), loader: 'expose?React' },
-      
-      // For React 0.14
-      { test: require.resolve('react-dom/server'), loader: 'expose?ReactDOMServer' }, // not in client one, only server
+```
+Usage:
+  rails generate react_on_rails:install [options]
+
+Options:
+  -d, [--redux], [--no-redux]                        # Setup Redux files
+  -S, [--server-rendering], [--no-server-rendering]  # Configure for server-side rendering of webpack JavaScript
+  -L, [--skip-linters], [--no-skip-linters]          # Don't install linter files
+
+Runtime options:
+  -f, [--force]                    # Overwrite files that already exist
+  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
+  -q, [--quiet], [--no-quiet]      # Suppress status output
+  -s, [--skip], [--no-skip]        # Skip files that already exist
+
+Description:
+    Create react on rails files for install generator.
 ```
 
+We have a repo showing the results of running the generator with various combinations of options, each combination on its own branch:
 
-#### Sample webpack.server.rails.config.js (ONLY for server rendering)
-Be sure to check out the latest example version of [client/webpack.server.rails.config.js](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.server.rails.config.js).
++ [basic (`rails generate react_on_rails:install`)](https://github.com/shakacode/react_on_rails-generator-results/tree/basic)
```

Comment:
Let's link to the PR actually, that shows the diff best of what the generate did.